progressEval.html

```diff
@@ -0,0 +1,458 @@
+      <input type="number" id="hw13" onchange="calculate()" value="" placeholder="Homework Week 13">
+      <input type="number" id="adv13" onchange="calculate()" value="" placeholder="Advanced Week 13">
+      <input type="number" id="ec13" onchange="calculate()" value="" placeholder="Extra Credit Week 13">
+    </p>
+    <p id="week14" style="visibility: hidden;">Week 14 
+      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 
+      <input type="number" id="att14" onchange="calculate()" value="" placeholder="Attendance Week 14">
+      <input type="number" id="hw14" onchange="calculate()" value="" placeholder="Homework Week 14">
+      <input type="number" id="adv14" onchange="calculate()" value="" placeholder="Advanced Week 14">
+      <input type="number" id="ec14" onchange="calculate()" value="" placeholder="Extra Credit Week 14">
+    </p>
+    <p id="week15" style="visibility: hidden;">Week 15 
+      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 
+      <input type="number" id="att15" onchange="calculate()" value="" placeholder="Attendance Week 15">
+      <input type="number" id="hw15" onchange="calculate()" value="" placeholder="No Homework Week 15" readonly="readonly">
+      <input type="number" id="adv15" onchange="calculate()" value="" placeholder="No Advanced Week 15" readonly="readonly">
+      <input type="number" id="ec15" onchange="calculate()" value="" placeholder="Extra Credit Week 15">
+    </p>
+    <p id="totals">Totals     &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+      <input type="number" id="attTot" value="" placeholder="Attendance Total" readonly="readonly">
+      <input type="number" id="hwTot"  value="" placeholder="Homework Total" readonly="readonly">
+      <input type="number" id="advTot" value="" placeholder="Advanced Total" readonly="readonly">
+      <input type="number" id="ecTot"  value="" placeholder="Extra Credit Total" readonly="readonly">
+    </p>
+    <p> Right now you have 
+      <input type="number" id="numPoints" value="" placeholder="Total" readonly="readonly">
+      points, which is a(n) 
+      <input type="text" id="curGrade"  value="" placeholder="A" readonly="readonly">
+    </p>
+    <p> If your point-to-week ratio stays constant, you will earn 
+      <input type="number" id="predPoints" value="" placeholder="Total" readonly="readonly">
+      points, which is a(n)
+      <input type="text" id="predGrade"  value="" placeholder="A" readonly="readonly">
+    </p>
+	</form>
+	
+	<script type="text/javascript">
+	function calculate(grades){
+	  var week = document.getElementById("weekselect").value || 0;
+      
+      var attTotal = 0;
+      var hwTotal = 0;
+      var advTotal = 0;
+      var ecTotal = 0;
+      
+      if(week == 0){
+      	document.getElementById("week0").style.visibility="visible";
+        document.getElementById("week0Alt").style.visibility="hidden";
```

Comment:
There's really no need to include this `if` clause if the dropdown menu just didn't include week 0. As in, if the only purpose of the week 0 option is to show that another week must be selected, why would week 0 be an option in the first place?